.msync.yml

```diff
@@ -2,4 +2,4 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
-modulesync_config_version: '9.5.0'
+modulesync_config_version: '10.0.0'
```

Comment:
should the subject / commit reflect 10.x or 9.6?

Reply:
@wyardley open moduleync PRs get update without changing the PR title